src/CodeGenerators/SettingsGen.Example/Testing.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Omex.CodeGenerators.SettingsGen.Models.Attributes;
+
+#nullable enable
+
+namespace Microsoft.Omex.SettingsGen.Example
+{
+	/// <summary>
+	/// 
+	/// </summary>
+	[Section(Name = "Renaming")]
+	public class Testing
+	{
+		/// <summary>
+		/// 
+		/// </summary>
+		public string Whatever { get; set; } = string.Empty;
+
+
+		/// <summary>
+		/// 
+		/// </summary>
+		[Parameter(Name = "TestingThis", Value = "Hmmmm")]
+		public string? DiffName { get; set; }
```

Comment:
Do these need to be nullable? The settings in the resulting xml have empty string values anyway.

Reply:
They don't need to be nullable and the settings.xml file will have the value in the value field

Reply:
What I meant here is if it needs to be "string?" and can't it just be "string DiffName"